stix2elevator/convert_stix.py

```diff
@@ -2228,9 +2289,10 @@ def handle_missing_properties_of_victim_target(identity_instance, victim_targeti
         if victim_targeting.targeted_information:
             handle_missing_string_property(container, "targeted_information",
                                            victim_targeting.targeted_information, identity_instance["id"], True)
-        if hasattr(victim_targeting, "technical_details") and victim_targeting.targeted_technical_details is not None:
-            handle_missing_string_property(container, "technical_details",
-                                           victim_targeting.targeted_technical_details, identity_instance["id"], True)
+        # TODO: technical_details are Observables
```

Comment:
Do we need to issue a message that this wasn't converted?

Reply:
added a warning message